README.md

```diff
@@ -32,3 +32,18 @@ benchopt run lasso -s sklearn -s baseline -d boston --max-samples 10 --repetitio
 ```
 
 Use `benchopt run -h` for more details about these options, or visit https://benchopt.github.io/api.html.
+
+
+
+List of optimization problems available
+---------------------------------------------------
```

Comment:
wait a sec it's a markdown file not an RST file. So use:

`### List of optimization problems available`